rules/no-await-in-promise-methods.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const isPromiseMethodWithArray = require('./utils/is-promise-method-with-array.js');
+
+const MESSAGE_ID = 'no-await-in-promise-methods';
+const messages = {
+	[MESSAGE_ID]: 'Parameters in `Promise.{{method}}` should not be awaited.',
+};
+const METHODS = ['all', 'allSettled', 'any', 'race'];
+
+const getArrayElements = node => node.arguments[0].elements;
+
+const getMethodName = node => node.callee.property.name;
+
+const getFixer = ({sourceCode}, element) => fixer => {
+	const {argument} = element;
+	const text = sourceCode.getText(argument);
+	const replacement = argument.type === 'SequenceExpression' ? `(${text})` : text;
+
+	return fixer.replaceText(element, replacement);
```

Comment:
Please remove `await` token instead of using `getText`.

Reply:
It is modified.